terraform/projects/app-ecs-services/templates/alertmanager.tpl

```diff
@@ -19,6 +19,10 @@ route:
   - receiver: "registers-zendesk"
     match:
       product: "registers"
+  - receiver: "re-observe-pagerduty"
+    match:
+      product: "grafana"
```

Comment:
I think the product should be `prometheus`

Reply:
This is now pushed up (we had changed it but it didnt get commited)